rbac/management/managers.py

```diff
@@ -66,3 +66,24 @@ def standard(self, tenant=None, tenant_id=None):
         """Delegate call to the WorkspaceQuerySet."""
         tenant_id = self._get_tenant_id(tenant=tenant, tenant_id=tenant_id)
         return self.get_queryset().standard(tenant_id)
+
+    def descendant_ids_with_parents(self, ids):
+        """Return the descendant and root workspace IDs based on roots supplied."""
+        with connection.cursor() as cursor:
+            sql = """
+                WITH RECURSIVE descendants AS
+                    (SELECT id,
+                            parent_id
+                    FROM management_workspace
+                    WHERE id = ANY(%s::uuid[])
```

Comment:
Should we include only standard workspaces ?

Reply:
We'll definitely want/need root and default. I haven't considered ungrouped yet too much, but curious what you're thinking the issue might be there?

Reply:
Based on conversation last week, we're going to leave this to include all.